addons/web/static/src/views/form/form_controller.scss

```diff
@@ -1154,6 +1154,24 @@
     }
 }
 
+.o_form_view.o_xxs_form_view.o_kanban_quick_create_form {
```

Comment:
this code should be in kanban (I would create a new file kanban_record_quick_create.scss next to [kanban_record_quick_create.xml](https://github.com/odoo/odoo/pull/175527/files#diff-16a82abc65adb5e8c21adb72850a62eee143f3532ae619ecaa04f9265d23c7e0))